docs/UserGuide.md

```diff
@@ -341,15 +438,34 @@ Format: `c-find KEYWORD [MORE_KEYWORDS] ...`
 * Employees matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`.
 
-Examples:
-* `c-find John` returns `john` and `John Doe`.
-* `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
+**:information_source: Note:** <br>
+The search is case-insensitive. e.g. `hans` will match `Hans`.<br>
+The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.<br>
+Only the name is searched..<br>
+Only full words will be matched e.g. `Han` will not match `Hans`.<br>
+Employees matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`.<br>
+
```

Comment:
Line 441~447 contains the same info with the section above (line 434~439). Maybe can delete 441~447.